client/my-sites/domains/domain-management/components/domain/cannot-update-contact-info.tsx

```diff
@@ -0,0 +1,16 @@
+import InfoNotice from './info-notice';
+import type { ResponseDomain } from 'calypso/lib/domains/types';
+
+const CannotUpdateContactInfo = ( {
+	redesigned,
+	domain,
+}: {
+	redesigned: boolean;
+	domain: ResponseDomain;
+} ) => {
+	return (
+		<InfoNotice redesigned={ redesigned } text={ domain.cannotUpdateContactInfoReason as string } />
+	);
+};
+
+export default CannotUpdateContactInfo;
```

Comment:
`CannotUpdateContactInfo` and `CannotManageDnsRecords` are the same and basically wrappers around `InfoNotice`, so I think maybe we should use `InfoNotice` directly in place of them 🙂

Reply:
You're totally right! I've refactored the code 😉